src/Controls/src/SourceGen/Controls.SourceGen.csproj

```diff
@@ -60,6 +60,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\BindingSourceGen\Controls.BindingSourceGen.csproj" />
+    <ProjectReference Include="..\..\..\Graphics\src\Graphics\Graphics.csproj" />
```

Comment:
nope

we don't want to have dependencies. can we replace that with a Compile-Include ?